how-to/customize-home-templates/public/contacts/1.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is this from a public free to use source? Also hoping that these are all fake details :)

Reply:
Yeah random generated

Reply:
link added in readme